Makefile

```
@@ -72,6 +72,10 @@ bin/$(BIN_NAME)-%.exe bin/$(BIN_NAME)-%: cmd/$(BIN_NAME) check_go_env
 bin/%: cmd/% check_go_env
 	$(GO_BUILD) -o $@$(EXEC_EXT) ./$<
 
+build-invocation-image:
+	@echo "Build invocation image if needed"
+	$(if $(shell docker images -q docker/cnab-app-base:$(TAG)),, make -f ./docker.Makefile invocation-image)
```

You can replace `make` by `$(MAKE)`.
And just for readability it can be multi lines.

```suggestion
	$(if $(shell docker images -q docker/cnab-app-base:$(TAG)),, \
	    $(MAKE) -f ./docker.Makefile invocation-image \
	)
```

---

Done, thanks 👍